.github/workflows/source_aws.yml

```diff
@@ -112,6 +112,7 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version-file: plugins/source/aws/go.mod
+          cache: false
```

Comment:
```suggestion
```

This already the default so not needed. We can do it once we update to v4

Reply:
I think it's better to leave this explicitly turned off as it doesn't affect the result now